bots/machine/testvm.py

```diff
@@ -627,9 +636,7 @@ def spawn(self, shell_cmd, log_id):
         return int(self.execute("{ (%s) >/var/log/%s 2>&1 & }; echo $!" % (shell_cmd, log_id)))
 
     def _calc_identity(self):
-        identity = os.path.join(BOTS_DIR, "machine", "identity")
-        os.chmod(identity, 0o600)
-        return identity
+        return self.identity_file
```

Comment:
Can you completely eliminate this helper function and just replace calls with `.identity_file`?